### PR TITLE
Adds basic overview

### PR DESCRIPTION
**Pull request by evilnick**

### CODE REVIEW

File: pages/k8s/overview.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+
+### Cloud integration
+
+Depending on your choice of cloud, **CDK** will also install an integration
+application. This enables **CDK** to seamlessly access cloud resources and
+components without additional configuration or hassle.
+
+
+
+<!-- LINKS -->>
```
Comment (degville) [inline pages/k8s/overview.md]: end comment typo

File: pages/k8s/overview.md [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+<p>Deliver ‘Containers as a Service’ across the enterprise with the
+<strong>Canonical Distribution of Kubernetes<sup>&reg;</sup> </strong>
+(<strong>CDK</strong>) , enabling each project to
+spin up a standardised K8s of arbitrary scale, on demand, with centralised
+operational control. <strong>CDK</strong>  provides a
+well integrated, turn-key <strong>Kubernetes<sup>&reg;</sup></strong>
+platform that is open, extensible, and secure.
+</p>
+<p>
+For more information on the features and benefits of **CDK**, including details
```
Comment (degville) [inline pages/k8s/overview.md]: This doesn't render bold for some reason.

File: pages/k8s/overview.md (hunk 1 above)
Comment (evilnick) [reply pages/k8s/overview.md]: d'oh!

File: pages/k8s/overview.md (hunk 2 above)
Comment (evilnick) [reply pages/k8s/overview.md]: yes, because it is in an html block. will fix that, thanks

**Pull request merged**